Automattic/wp-calypso
# Add order history to campaign overview page

Related to  995-gh-tumblr/a8c-dsp

## Proposed Changes

* Adds order history to the campaign overview page:

## Testing Instructions
- Load the live calypso link from the comments below
- Visit `/advertising`
- Select a campaign that you have completed orders for.
    -  tumblr shop has plenty if you are short of your own `/advertising/campaigns/63282/shop.tumblr.com`
- You should see the order breakdown ( this is for any campaign type, not just evergreen)


![Screenshot 2024-03-20 at 12 09 30](https://github.com/Automattic/wp-calypso/assets/6440498/7379bfed-6686-4a12-abe5-1ed0426384a4)

## Pre-merge Checklist
